scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -118,6 +116,41 @@ public static BuildTargetResult PublishDebFileToDebianRepo(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [Environment("DOCKER_HUB_REPO")]
```

Comment:
We're controlling which CLI builds trigger DockerHub builds through the use of VSTS build variables.  I think it's better to control it in one place rather than in two places if possible.  If we add a CentOS Docker image in addition to our Ubuntu image, for example, I think it would be better to be able to just add the build variables to the CentOS build definition rather than also needing to add a target condition allowing it to run on CentOS.

Not having the Ubuntu condition also makes it easier to test locally.